.devcontainer/scripts/keycloak-local.js

```diff
@@ -0,0 +1,194 @@
+require('dotenv').config();
+const KcAdminClient = require('@keycloak/keycloak-admin-client').default;
```

Comment:
Do you know why it's ignoring this file?

Reply:
I have no clue. I tried searching why but no luck.